### PR TITLE
feat(reset): add sanitize.css

### PR DESCRIPTION
**Pull request by johannschopplich**

### CODE REVIEW

File: packages/reset/scripts/copy.ts [hunk 1]
```diff
@@ -4,3 +4,20 @@ fs.copyFileSync(
...
+  'forms.css',
+  'assets.css',
+  'typography.css',
+  'reduce-motion.css',
+  'system-ui.css',
+  'ui-monospace.css',
+]) {
+  fs.copyFileSync(
+    `node_modules/sanitize.css/${stylesheet}`,
+    `sanitize.css/${stylesheet}`,
```
Comment (antfu) [inline packages/reset/scripts/copy.ts:21]:
```suggestion
    `sanitize/${stylesheet}`,
```

File: packages/reset/sanitize.css/assets.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/**
```
Comment (antfu) [inline packages/reset/sanitize.css/assets.css:1]: I guess we'd better exclude them from the source control.

File: packages/reset/scripts/copy.ts (hunk 1 above)
Comment (johannschopplich) [reply packages/reset/scripts/copy.ts]: Good idea.

File: packages/reset/sanitize.css/assets.css (hunk 2 above)
Comment (johannschopplich) [reply packages/reset/sanitize.css/assets.css]: True. 🙂

Comment (johannschopplich) [reply packages/reset/sanitize.css/assets.css]: Have pushed a commit addressing your suggestion.

**Pull request merged**